Makefile

```diff
@@ -200,7 +201,7 @@ test:
 .PHONY: test-verbose
 test-verbose: build-tests
 test-verbose:
-	@PYTHONFAULTHANDLER=1 python -m pytest --ignore ./tests/docker -vv -s ./tests
+	PYTHONFAULTHANDLER=1 python -m pytest $(COV_FLAGS) --ignore ./tests/docker -vv -s ./tests
```

Comment:
You never actually set COV_FLAGS to anything other than an empty string. Did you mean to come back to this?

Reply:
As discussed offline, this gets passed in via the command line call in `run_tests.yml`